packages/core/src/components/group/group.tsx

```diff
@@ -184,7 +193,16 @@ export class Group {
     }
   }
 
+  private observer: MutationObserver;
+
   componentDidLoad() {
+    const slotDiv = this.hostElement.querySelector('.group-content');
+    this.observer = createMutationObserver(() => {
+      this.slotSize = this.groupItems.length;
```

Comment:
slotSize is nerver used again in the component? Do you have any local changes which are not pushed?